cmd/eventlistenersink/main.go

```diff
@@ -83,7 +88,29 @@ func main() {
 	profilingServer := profiling.NewServer(profilingHandler)
 	metrics.MemStatsOrDie(ctx)
 
-	sharedmain.WatchObservabilityConfigOrDie(ctx, configMapWatcher, profilingHandler, logger, EventListenerLogKey)
+	metricsPort, err := strconv.ParseInt(os.Getenv("METRICS_PROMETHEUS_PORT"), 10, 64)
+	if err != nil {
```

Comment:
Do we still need all of this code if we set the METRICS_PROMETHEUS_PORT? My understanding was that the metrics code looks for this env var and if it is set, it will automatically use that port (just like we do with METRICS_DOMAIN). 

See: https://github.com/tektoncd/triggers/blob/4939226120cdf4646e1f5941fb1b10b8f866abac/vendor/knative.dev/pkg/metrics/config.go#L308-L325

Reply:
Yes you are right 
Forgot to remove the code after setting env

Updated code now